ports/libsbml/portfile.cmake

```
@@ -1,31 +1,30 @@
-vcpkg_download_distfile(ARCHIVE
-    URLS "https://sourceforge.net/projects/sbml/files/libsbml/5.18.0/stable/libSBML-5.18.0-core-plus-packages-src.tar.gz/download"
-    FILENAME "libSBML-5.18.0.zip"
-    SHA512 49dedaa2fcd2077e7389a8f940adf931d80aa7a8f9d57330328372d2ac8ebcaeb03a20524df2fe0f1c6933587904613754585076c46e6cb5d6f7a001f427185b
-)
+vcpkg_fail_port_install(ON_TARGET "uwp")
 
-vcpkg_extract_source_archive_ex(
+vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    ARCHIVE ${ARCHIVE}
-    PATCHES fix-linkage-type.patch
+    REPO sbmlteam/libsbml
+    REF 118ffbf11f1a5245cc544c1eac71019d979ecb20 #libSBML-5.19.0
+    SHA512 7fe8b4d594876c6408e01c646187cb1587d0b4e12707a43286150d4e4646841e547bde971de917de1cdfbbb9365172aeac43c8e02f7d354400f9166f0f1c2c3d
+    HEAD_REF stable

```

```suggestion
    HEAD_REF development
```
The stable branch should always lag behind the development branch.